packages/components/pie-modal/README.md

```diff
@@ -72,6 +72,7 @@ import { PieModal } from '@justeattakeaway/pie-modal/dist/react';
 | isLoading | `Boolean` | `false` | When true, displays a loading spinner in the modal |
 | size | `String` | `medium` | Determines the maximum width of the modal. Large modals will expand to fill the entire page width at narrow viewports. Can be `small`, `medium` or `large`. |
 | returnFocusAfterCloseSelector | `String` | `undefined` | If provided, focus will be sent to the first element that matches this selector when the modal is closed. If not provided, the `dialog` element will return focus to the element that opened the modal. |
+| leadingAction | `Object` | `undefined` | An object representing the leading action of the modal. It has the following properties:<br>- text (required): The text to display on the leading action button.<br>- variant: The variant of the leading action button. It can be one of primary, secondary, ghost, or ghost-secondary. Defaults to primary if not provided.<br>- ariaLabel: The ARIA label for the leading action button. |
```

Comment:
Its worth bearing in mind we will need to manually keep this up to date with the button variant type in this readme. I won't if a separate prop table would be useful for object properties like this. Otherwise it's quite easy to miss. Eventually I think we'd want to automate this

Reply:
I think it would be better to just link to the prop table for pie-button here:

I've added a suggestion below